SPEC.md

```diff
@@ -455,34 +590,71 @@ lines](https://www.unicode.org/versions/Unicode13.0.0/ch05.pdf):
 
 Note that for the purpose of new lines, CRLF is considered _a single newline_.
 
+### Disallowed Literal Code Points
+
+The following code points may not appear literally anywhere in the document.
+They may be represented in Strings (but not Raw Strings) using `\u{}`.
+
+* Any codepoint with hexadecimal value `0x20` or below (various control
```

Comment:
0x20 is space, so this seems to imply that space is not allowed inside quoted strings.

Also, should BOM be included here? (Otherwise, I guess, there could be a bit of conflict of how `<U+FEFF>node` at the beginning of the document should be parsed. Disambiguating by longest token, it is parsed as one identifier.)

Reply:
The "0x20 or below rule" also includes `0x09` (the tab `\t`), `0x0A` (newline `\n`), `\x0B` (vertical tab `\v`), `0x0C` (form feed `\f`), and `0x0D` (carriage return `\r`).

These are all considered either newline or whitespace in this spec and should be excluded from this rule.

Reply:
fixed